android/src/main/java/com/zebralinkos/ZebraLinkOsModule.java

```diff
@@ -0,0 +1,32 @@
+package com.zebralinkos;
+
+import androidx.annotation.NonNull;
+
+import com.facebook.react.bridge.Promise;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.facebook.react.bridge.ReactMethod;
+import com.facebook.react.module.annotations.ReactModule;
+
+@ReactModule(name = ZebraLinkOsModule.NAME)
+public class ZebraLinkOsModule extends ReactContextBaseJavaModule {
+  public static final String NAME = "ZebraLinkOs";
+
+  public ZebraLinkOsModule(ReactApplicationContext reactContext) {
```

Comment:
Is this (or any other android) file intended?

Reply:
Removed everything related to Android